test/navigation/FundRaisingNavigatorSpec.scala

```diff
@@ -56,10 +56,32 @@ class FundRaisingNavigatorSpec extends SpecBase {
         "go to Has your charity prepared financial accounts page when clicked continue button" in {
           navigator.nextPage(OperatingLocationPage, NormalMode,
             emptyUserAnswers.set(OperatingLocationPage, Set[OperatingLocationOptions](OperatingLocationOptions.Overseas)).getOrElse(emptyUserAnswers)) mustBe
-            routes.IndexController.onPageLoad() // TODO modify once prepared financial accounts page created
+            operationFundsRoutes.IsFinancialAccountsController.onPageLoad(NormalMode)
         }
       }
 
+      "from the IsFinancialAccountsPage" must {
+
+        "go to the SessionExpiredController page when user answer is empty" in {
+          navigator.nextPage(IsFinancialAccountsPage, NormalMode, emptyUserAnswers) mustBe
+            routes.SessionExpiredController.onPageLoad()
+        }
+
+        "go to the Index Controller page when yes is selected" in {
+
+          navigator.nextPage(IsFinancialAccountsPage, NormalMode,
+            emptyUserAnswers.set(IsFinancialAccountsPage,true).success.value) mustBe
+            routes.IndexController.onPageLoad()
+        }
+        "go to the Governing Document summary page when no is selected" in {
```

Comment:
could you please update this with correct page name in test label

Reply:
Updated and pushed it to the branch